packages/core/src/app/shipping/ShippingFormFooter.tsx

```diff
@@ -29,13 +29,13 @@ const ShippingFormFooter: FunctionComponent<ShippingFormFooterProps> = ({
     isLoading,
 }) => {
     const { extensionService, isExtensionEnabled } = useExtensions();
-    const isRegionInUse = Boolean(
+    const isExtensionRegionEnabled = Boolean(
```

Comment:
The real purpose of this boolean flag is to determine whether or not we render the container and extension, so I renamed it.